data/data/openstack/topology/sg-worker.tf

```
@@ -17,13 +17,12 @@ resource "openstack_networking_secgroup_rule_v2" "worker_ingress_icmp" {
 }
 
 resource "openstack_networking_secgroup_rule_v2" "worker_ingress_ssh" {
-  direction      = "ingress"
-  ethertype      = "IPv4"
-  protocol       = "tcp"
-  port_range_min = 22
-  port_range_max = 22
-  # FIXME(mandre) AWS only allows SSH from cidr_block
-  remote_ip_prefix  = "0.0.0.0/0"
+  direction        = "ingress"
```

The missing space makes [ci/prow/tf-fmt angry](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/4659/pull-ci-openshift-installer-master-tf-fmt/1361671440692154368/build-log.txt):
```suggestion
  direction         = "ingress"
```